sources/packages/forms/src/form-definitions/studentdependantsappealparttime.json

```diff
@@ -0,0 +1,970 @@
+                                                    "type": "htmlelement",
+                                                    "hideLabel": false,
+                                                    "tags": [],
+                                                    "conditional": {
+                                                        "show": "",
+                                                        "when": null,
+                                                        "eq": ""
+                                                    },
+                                                    "properties": {},
+                                                    "customClass": "align-bullets",
+                                                    "lockKey": true,
+                                                    "isNew": false
+                                                },
+                                                {
+                                                    "autofocus": false,
+                                                    "input": true,
+                                                    "tableView": false,
+                                                    "label": "PD Dependent",
+                                                    "key": "pdDependentUpload",
+                                                    "image": false,
+                                                    "imageSize": "200",
+                                                    "placeholder": "",
+                                                    "multiple": true,
+                                                    "defaultValue": "",
+                                                    "protected": false,
+                                                    "persistent": true,
+                                                    "hidden": false,
+                                                    "clearOnHide": true,
+                                                    "filePattern": ".pdf,.doc,.docx,.jpg,.png,.txt",
+                                                    "fileMinSize": "0KB",
+                                                    "fileMaxSize": "4MB",
+                                                    "type": "file",
+                                                    "labelPosition": "top",
+                                                    "tags": [],
+                                                    "conditional": {
+                                                        "show": "true",
+                                                        "when": "declaredOnTaxes",
+                                                        "eq": "yes"
+                                                    },
+                                                    "properties": {},
+                                                    "storage": "url",
+                                                    "url": "student/files",
```

Comment:
Are we suppose to re-use the student files table for appeals??

Reply:
As discussed in the meeting, we will keep the same url but add "Appeal" as an enum type to the file_origin column.